classes/class-kco-api.php

```diff
@@ -158,7 +158,7 @@ public function update_klarna_confirmation( $klarna_order_id, $klarna_order, $or
 	/**
 	 * Checks for WP Errors and returns either the response as array or a false.
 	 *
-	 * @param array $response The response from the request.
+	 * @param array|WP_Error $response The response from the request.
```

Comment:
Nitpick: This is incorrect now right? Since it will only return an array or bool value.

Reply:
I think you meant to refer to the return value, not the param :grin:

I've updated the docblock to return `array|false`.